hw2/04-background.js

```diff
@@ -1 +1,62 @@
-// Add your code here
+document.addEventListener("DOMContentLoaded", function () {
+    let intervalId;
+    let isRunning = false;
+    let intervalSeconds = 3;
+    //const fixedColors = ['red', 'green', 'blue', 'yellow', 'purple', 'cyan'];
+    const fixedColors = [
+        "hsla(278, 35%, 18%, 0.5)",
+    "hsla(337, 100%, 29%, 0.5)",
+    "hsla(338, 100%, 24%, 0.5)",
+    "hsla(208, 97%, 27%, 0.5)",
+    "hsla(209, 100%, 22%, 0.5)",
+      ];
+    let currentColorIndex = 0;
+
+    function setBackgroundColor(color) {
+        console.log(`Setting background color to ${color}`);
+        document.body.style.backgroundColor = color;
+    }
+
+    function startBackgroundColorChange() {
+        console.log("Starting background color change...");
+        isRunning = true;
+        intervalId = setInterval(() => {
+            const color = fixedColors[currentColorIndex];
+            setBackgroundColor(color);
+            console.log(`Background color changed to ${color}`);
+            currentColorIndex = (currentColorIndex + 1) % fixedColors.length; 
+        }, intervalSeconds * 1000);
```

Comment:
This might sound a little nitpicky, sorry about that.
I feel since intervalSeconds is being has a value being provided in the code, I think including an error handling to check if this is a valid value would help. :)